src/test/java/org/javarosa/core/model/TriggerableDagTest.java

```diff
@@ -372,7 +382,6 @@ public void verify_relation_between_calculate_expressions_and_relevancy_conditio
         // and number1_x2_x2 doesn't get evaluated. The difference could be that the second field depends
```

Comment:
I think this comment can now be updated. We now know that even though calculations in non-relevant groups are evaluated, their values are unavailable to other parts of the form. This is the same deviation as https://github.com/opendatakit/javarosa/pull/543/files#diff-fbf45323aa2e937c91c75ba91cad6ebdR983.